evalml/automl/automl_search.py

```diff
@@ -673,16 +673,19 @@ def _compute_cv_scores(self, pipeline, X, y):
     def _add_result(self, trained_pipeline, parameters, training_time, cv_data, cv_scores):
         cv_score = cv_scores.mean()
         percent_better = self.objective.calculate_percent_difference(cv_score, self._baseline_cv_score)
-        # calculate high_variance_cv
-        # if the coefficient of variance is greater than .2
-        with warnings.catch_warnings():
-            warnings.simplefilter('ignore')
-            high_variance_cv = (cv_scores.std() / cv_scores.mean()) > .2
 
         pipeline_name = trained_pipeline.name
         pipeline_summary = trained_pipeline.summary
         pipeline_id = len(self._results['pipeline_results'])
 
+        high_variance_cv_check = HighVarianceCVDataCheck(threshold=0.2)
```

Comment:
Should users be allowed to configure this threshold now that this is a DataCheck and we let users configure other data checks?

Reply:
good thought! Perhaps we can turn this on and off depending on if `data_checks = auto` vs. `data_checks = disabled`. But I don't think it'll fit within the existing API for parameterizing data checks as all those data checks run before search is called whereas this check is called during search. I like the idea but we would need to think about what API changes to make to `AutoMLSearch.search()`.

Reply:
👍  yep @freddyaboulton I agree this should be configurable/disable-able.

This PR is essentially porting existing behavior into a new API (data checks). I'll file an issue now to track making this configurable.